packages/paste-core/components/tabs/src/Tab.tsx

```diff
@@ -56,39 +56,60 @@ const getTabBoxStyles = (orientation: Orientation, variant: Variants): BoxStyleP
     case 'horizontal':
     default:
       return {
-        borderBottomColor: 'transparent',
-        borderBottomStyle: 'solid',
-        borderBottomWidth: 'borderWidth10',
-        color: isInverse ? 'colorTextInverseWeak' : 'colorTextWeak',
+        color: isInverse ? 'colorTextInverse' : 'colorTextWeak',
+        borderColor: 'transparent',
+        borderStyle: 'solid',
+        borderWidth: 'borderWidth10',
+        borderTopLeftRadius: 'borderRadius30',
+        borderTopRightRadius: 'borderRadius30',
         display: 'inline-block',
         flexBasis: isFitted ? '50%' : undefined,
         flexGrow: isFitted ? 1 : undefined,
         flexShrink: isFitted ? 1 : undefined,
-        marginRight: isFitted ? 'space0' : 'space80',
         minWidth: 'sizeSquare130',
-        paddingBottom: 'space50',
-        paddingX: 'space20',
-        paddingTop: 'space40',
+        paddingY: 'space30',
+        paddingX: 'space50',
         textAlign: 'center',
-        _last: {
-          marginRight: 'space0',
-        },
+        bottom: '-1px',
+        /* Selected State */
         _selected: {
-          borderBottomColor: isInverse ? 'colorBorderInverseStrong' : 'colorBorderPrimary',
-          color: isInverse ? 'colorTextInverse' : 'colorTextLink',
-        },
-        _hover: {
-          borderBottomColor: isInverse ? 'colorBorderInverse' : 'colorBorderPrimaryStronger',
-          color: isInverse ? 'colorTextInverseWeak' : 'colorTextLinkStronger',
+          borderColor: isInverse ? 'colorBorderInverseStrongest' : 'colorBorderWeak',
+          borderBottomColor: isInverse ? 'colorBorderInverseWeakest' : 'colorBorderWeakest',
+          color: isInverse ? 'colorTextInverse' : 'colorTextPrimary',
         },
+        /* Disabled state */
         _disabled: {
           borderBottomColor: 'transparent',
           color: isInverse ? 'colorTextInverseWeaker' : 'colorTextWeaker',
         },
+        /* Hover states */
+        _hover: {
+          color: isInverse ? 'colorTextInverse' : 'colorTextPrimary',
+          borderBottomColor: isInverse ? 'colorBorderInverseStrongest' : 'colorBorderPrimary',
```

Comment:
on hover:
- in default, the top/right/left border color stays the same (color-border-weak)
- in inverse, the top/right/left border color changes to color-border-inverse-strongest